mobile/views/components/index_photo_list.html

```diff
@@ -0,0 +1,56 @@
+{style}{literal}
+.ow_photo_item_widget {
+    width: 72px;
+    height: 72px;
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+{/literal}{/style}
+{capture assign='nocontent'}
+    <div class="owm_nocontent">{text key='photo+no_photo'}</div>
+{/capture}
+
+{capture assign='cmp'}
+    <div id="photo_list_cmp{$uniqId}">
+        <div class="owm_index_photo_list" id="{$items.latest.contId}">
+            {foreach from=$latest item='p'}     
+                <div class="owm_index_photo_item{if !empty($data.class)} {$data.class}{/if}">
```

Comment:
Could not find where you assigned `$data` to template. It seems that the variable is `null`